middleware/middleware-1.3.14.md

```diff
@@ -0,0 +1,80 @@
+---
+slug: /release-notes/middleware/1.3.14
+title: Version 1.3.14
+---
+
+# fiskaltrust.Middleware 1.3.14 (Germany)
+_February 17, 2021_
+
+This quality-of-life update for the Middleware fixes multiple bugs, introduces stability improvements, and adds CORS support to our Queues.
```

Comment:
From a technical perspective cors support is added to the Launcher. In fact you could also use the SCU via cors

Reply:
Removed this paragraph